### PR TITLE
fix: Enable CORS & Minor Changes

### PR DESCRIPTION
**Pull request by CIPHERTron**

### CODE REVIEW

File: server/routes/user.py [hunk 1]
```diff
@@ -50,14 +53,15 @@ def login():
             },
             app.config["SECRET_KEY"],
         )
-        return jsonify({"token": token})
+    return jsonify({"token": token, "message": "Login Successful!"})
```
Comment (JPTIZ) [inline server/routes/user.py:56]: Was this change intended? :thinking: Because if the code doesn't enter the `if` block, `token` won't be defined and the function will mostly like break and return as a 500.

**Review by JPTIZ — COMMENTED** [review]
Overall looks fine, just check my comment there.

File: server/routes/user.py (hunk 1 above)
Comment (CIPHERTron) [reply server/routes/user.py:56]: Ohh yes....I guess indentation error 
My bad 😅 
I'll make the required changes 🚀

**Review by JPTIZ — APPROVED** [review]
